contrib/removed/finufft.h

```diff
@@ -0,0 +1,22 @@
+#ifndef FINUFFT_H
```

Comment:
Why are we keeping this file?

Reply:
This is again a mistake. Files under removed wasn't intended to be included....